### PR TITLE
Add protvista uniprot

### PR DESCRIPTION
**Pull request by aurel-l**
PR just to keep track of attempt of using ProtVista within the new website.

### CODE REVIEW

File: src/entry/Entry.tsx [hunk 1]
```diff
@@ -162,6 +164,12 @@ const Entry: React.FC<EntryProps> = ({
       ),
       mainContent: <EntryMain transformedData={entryData} />,
     },
+    {
+      name: 'Feature viewer',
+      path: 'feature-viewer',
+      icon: <ProteinIcon />,
```
Comment (aurel-l) [inline src/entry/Entry.tsx:170]: Using the protein icon for now (same one than the loading component), awaiting better one from Austra

File: src/entry/FeatureViewer.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  // // not false, otherwise will render as "false" in the DOM, we need undefined
+  // const [suspended, setSuspended] = useState<true | undefined>(undefined);
+  // const data = useDataApi(
+  //   `https://www.ebi.ac.uk/uniprot/api/covid-19/uniprotkb/accession/${accession}`
+  // );
+  const data = useDataApi(
+    `https://www.ebi.ac.uk/proteins/api/proteins/${accession}`
+  );
+
+  // useEffect(() => {
```
Comment (aurel-l) [inline src/entry/FeatureViewer.tsx:28]: At the moment, just let the normal component logic flow. Eventually we want to pause the logic with "suspend" attribute and inject data here.

File: src/entry/FeatureViewer.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+const FeatureViewer: FC<{ accession: string }> = ({ accession }) => {
+  // const viewerRef: any = useRef();
+  // // not false, otherwise will render as "false" in the DOM, we need undefined
+  // const [suspended, setSuspended] = useState<true | undefined>(undefined);
+  // const data = useDataApi(
+  //   `https://www.ebi.ac.uk/uniprot/api/covid-19/uniprotkb/accession/${accession}`
+  // );
+  const data = useDataApi(
+    `https://www.ebi.ac.uk/proteins/api/proteins/${accession}`
```
Comment (aurel-l) [inline src/entry/FeatureViewer.tsx:25]: Just getting data from the main UniProt API to make sure the entry exists there before even trying to load the ProtVista visualisation

File: src/entry/FeatureViewer.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  //   };
+  // }, [data]);
+
+  if (!data) return null;
+
+  return (
+    <section>
+      <div style={{ height: '80vh' }}>
+        <h2>ProtVista visualisation for {accession}</h2>
+        <p>
```
Comment (xwatkins) [inline src/entry/FeatureViewer.tsx:56]: So it turns out you can remove that ;-)

Comment (aurel-l) [reply src/entry/FeatureViewer.tsx:56]: Do you mean the disclaimer text about out-of-sync data?

**Pull request merged**